### PR TITLE
add arm64 builds

### PR DESCRIPTION
**Pull request by 0xbe7a**
- add arm64
- MNT: Re-rendered with conda-build 3.28.2, conda-smithy 3.30.4, and conda-forge-pinning 2024.02.13.16.11.45
- bump build

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -32,11 +32,13 @@ requirements:
...
     - make  # [linux]
     - pkg-config  # [linux]
+    - python                                 # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
   host:
     - pip
     - python
   run:
-    - importlib-metadata >=2.0,<3  # [py<38]
+    - importlib-metadata >=2.0,<3
```
Comment (pavelzw) [inline recipe/meta.yaml]:
```suggestion
```

Comment (0xbe7a) [reply recipe/meta.yaml]: i already removed it

Comment (pavelzw) [reply recipe/meta.yaml]: no i meant remove the whole line. it's not necessary with python >=3.8

**Comment by 0xbe7a** [comment]
@raybellwaves this is ready for review

**Pull request merged**